### ISSUE TITLE
0013862:
CalDAV and CardDAV sync fail on PROPFIND - HTTP 404

### ISSUE THREAD

**Issue by Gloirin** [tine20/tine20#6855]
**Reported by lhitc on 25 May 2018 13:39**

**Version:** 2018.02.4 Community Edition

This is actually about 2018.02.5 Community Edition.
I use a Debian 9 VM as server, Apache web server.
Client is DAVdroid on an Android 8 phone.

Instantly after installing 2018.02.5, my Android phone started showing sync errors.

The following shows up in Apache&#039;s accesslog (same for addressbooks):

PROPFIND /tine20/calendars/[SOME_LONG_NUMBER]/[SOME_SHORT_NUMBER]/ HTTP/1.1&quot; 404 335 &quot;-&quot; &quot;DAVdroid/1.11.3-gplay (2018/05/10; dav4android; okhttp/3.10.0) Android/8.0.0&quot;

Sync worked perfectly before the update.
The update itself showed no errors.

**Steps to reproduce:** - Have Tine20 2018.02.4 CE on the server
- Install DAVdroid on an Android phone, create an account and make sure it syncs
- Upgrade to Tine20 2018.02.4 CE
- try a sync on the phone

**Comment by Gloirin** [comment]
**Comment posted by gio32 on 4 Jun 2018 17:18**

After https://github.com/tine20/Tine-2.0-Open-Source-Groupware-and-CRM/issues/77 I&#039;m now stuck at this issue as  well - there&#039;s no meaningful logentry except for the 404 for PROPFIND on calendars

**Comment by wAmpIre** [comment]
Just updated to 2018.02.6 -- but bug remains.

This bug renders Tine20 completely unusable for me. And I don't think I'm the only one syncing Tine20 with *DAV. Please fix this bug ASAP!

**Comment by pschuele** [comment]
bug is already fixed - see #96 
we'll publish a new release on thursday!
closing this as duplicate.

**Comment by gio32** [comment]
I don't believe that this issue and #96 are identical, at least I don't have a Debug-Level dependent problem. Installing 2018.02.6 didn't help.
Here's what I see:
9b47c -- none -- - 2018-06-15T13:34:05+00:00 INFO (6): Tinebase_Core::setupLogger::796 Logger initialized.
9b47c -- none -- - 2018-06-15T13:34:05+00:00 INFO (6): index.php (20) Start processing request (PID: 25565)
9b47c -- none -- - 2018-06-15T13:34:05+00:00 DEBUG (7): Tinebase_Core::setupCache::848 Cache logging disabled
9b47c -- none -- - 2018-06-15T13:34:05+00:00 INFO (6): Tinebase_Core::setupCache::917 cache of backend type 'File' enabled
9b47c -- none -- - 2018-06-15T13:34:05+00:00 DEBUG (7): Tinebase_Config_Abstract::_loadAllAppConfigsInCache::607 Loading all configs for app Tinebase
9b47c -- none -- - 2018-06-15T13:34:05+00:00 DEBUG (7): Tinebase_Core::createAndConfigureDbAdapter::1038 Creating Pdo_Mysql DB adapter (db name: tine20)
9b47c -- none -- - 2018-06-15T13:34:05+00:00 DEBUG (7): Tinebase_Config_Abstract::_loadAllAppConfigsInCache::625 Found 20 configs.
9b47c -- none -- - 2018-06-15T13:34:05+00:00 INFO (6): Tinebase_Core::initFramework::455 initializing framework (PID: 25565)
9b47c -- none -- - 2018-06-15T13:34:05+00:00 DEBUG (7): Tinebase_Core::setupUserLocale::1192 given localeString 'auto'
9b47c -- none -- - 2018-06-15T13:34:05+00:00 DEBUG (7): Tinebase_Core::setupUserLocale::1219 Try to detect the locale of the user (browser, environment, default)
9b47c -- none -- - 2018-06-15T13:34:05+00:00 DEBUG (7): Tinebase_Translation::getLocale::199 given localeString 'auto'
9b47c -- none -- - 2018-06-15T13:34:05+00:00 DEBUG (7): Tinebase_Translation::getLocale::212 'de_DE' is not supported, checking fallback
9b47c -- none -- - 2018-06-15T13:34:05+00:00 DEBUG (7): Tinebase_Translation::getLocale::238 selected locale: 'de'
9b47c -- none -- - 2018-06-15T13:34:05+00:00 INFO (6): Tinebase_Core::setupUserLocale::1243 Setting user locale: de
9b47c -- none -- - 2018-06-15T13:34:05+00:00 DEBUG (7): Tinebase_Core::setupUserLocale::1250 Setting CTYPE locale from "C" to "en_US.UTF8".
9b47c -- none -- - 2018-06-15T13:34:05+00:00 INFO (6): Tinebase_Server_Expressive::handle::90 Is Routing request. uri: /tine20/calendars/c94ea67d249021fe9c00b629dcab352e6016e39a/19/?doRouting=1
9b47c -- none -- - 2018-06-15T13:34:05+00:00 DEBUG (7): Tinebase_Expressive_Middleware_ResponseEnvelop::process::36 processing...
9b47c -- none -- - 2018-06-15T13:34:05+00:00 DEBUG (7): Tinebase_Expressive_Middleware_FastRoute::process::40 processing...
9b47c -- none -- - 2018-06-15T13:34:05+00:00 DEBUG (7): Tinebase_Expressive_Middleware_FastRoute::process::54 FastRoute dispatching on method: PROPFIND and uri: /calendars/c94ea67d249021fe9c00b629dcab352e6016e39a/19
9b47c -- none -- - 2018-06-15T13:34:05+00:00 INFO (6): Tinebase_Expressive_Middleware_FastRoute::process::61 returning 404 not found
9b47c -- none -- - 2018-06-15T13:34:05+00:00 DEBUG (7): Tinebase_Expressive_Middleware_ResponseEnvelop::process::41 inspecting response...
9b47c -- none -- - 2018-06-15T13:34:05+00:00 INFO (6): index.php (31) METHOD: Tinebase_Server_Expressive:: / TIME: 107ms /  Memory usage: 2 MB /  Real patch cache size: 0 / PID: 25565

Those FastRoute dispatchings for PROPFIND methods always fail on calendars and address bocs via CardDAV and CalDAV on my installation since I updated to 2018.02.5, and 2018.02.6 didn't change the behaviour. I tried digging deeper but cannot yet provide any additional possibly useful info.
